docs/src/references/c8y-firmware-management.md

```diff
@@ -0,0 +1,214 @@
+# Device Firmware Management using Cumulocity
+
+Thin-edge provides an operation plugin to
+[manage device firmware using Cumulocity](https://cumulocity.com/guides/users-guide/device-management/#firmware-repo).
+Firmware management is currently supported only for child devices and not for the main tedge device.
+
+## Installation
+
+The plugin will be installed at `/usr/bin/c8y-firmware-plugin` by the debian package.
+The systemd service definition files for the plugin are also installed at `/lib/systemd/system/c8y-firmware-plugin.service`.
+
+No operations files are created under `/etc/tedge/operations/c8y/`
+as this plugin doesn't support firmware updates for the tedge device.
+Operation files for child devices must be created as part of their bootstrap process, which is explained later.
+
+## Configuration
+
+The plugin supports a single tedge configuration named `firmware.child.update.timeout`,
+that defines the amount of time the plugin wait for a child device to finish a firmware update once the request is delivered.
+The default timeout value is `3600s` and can be updated with:
```

Comment:
Does this mean the tedge config command would accept the value `3600s`, or is the unit of the config just in seconds, but only the value of `3600` would be accepted. For example:

```
sudo tedge config set firmware.child.update.timeout 3600s
```

If `3600s` is not accepted as a valid value then it would be better to be explicit about it.

```suggestion
The default timeout (in seconds) value is `3600` and can be updated with:
```